plugins/portals-list.js

```diff
@@ -257,9 +305,19 @@ window.plugin.portalslist.portalTable = function(sortBy, sortOrder, filter) {
 
   if(filter !== 0) {
     portals = portals.filter(function(obj) {
+      if (Math.abs(filter) <= 3) {
       return filter < 0
         ? obj.portal.options.team+1 != -filter
         : obj.portal.options.team+1 == filter;
+      }
+      switch (filter) {
+        case 4: {return (obj.portal.options.data.history.visited)};
+        case 5: {return (obj.portal.options.data.history.captured)};
+        case 6: {return (obj.portal.options.data.history.scoutControlled)};
+        case -4: {return (!obj.portal.options.data.history.visited)};
+        case -5: {return (!obj.portal.options.data.history.captured)};
+        case -6: {return (!obj.portal.options.data.history.scoutControlled)};
```

Comment:
braces and parenthesis are unneeded

Reply:
when single line if-clauses should have braces, case should have as well.

Reply:
that's true for if/else because of ambiguity, it's not for cases that are not ambiguous

Reply:
```suggestion
        case 4:
          return obj.portal.options.data.history.visited;
        case 5:
          return obj.portal.options.data.history.captured;
        case 6:
          return obj.portal.options.data.history.scoutControlled;
        case -4:
          return !obj.portal.options.data.history.visited;
        case -5:
          return !obj.portal.options.data.history.captured;
        case -6:
          return !obj.portal.options.data.history.scoutControlled;
```

Reply:
done